python-package/xgboost/spark/core.py

```diff
@@ -257,21 +258,20 @@ class _SparkXGBParams(
         "launched on the driver side; otherwise, it will be launched on the executor side.",
         TypeConverters.toBoolean,
     )
-    tracker_host_ip = Param(
+    tracker = Param(
```

Comment:
the name "tracker" is ok?

Reply:
I would probably continue to use the name `coll_cfg` just to be consistent. Besides, the configuration is for both the tracker and the communicators.

Reply:
sure, I changed it to `collective_conf`

Reply:
I know everyone has a naming preference ...  if you use a new name, please modify the existing ones in Dask for consistency.

Reply:
OK, I just changed it to coll_cfg to be compatible with dask.